README.md

```diff
@@ -17,83 +17,87 @@
 
 ## Prerequisites
 
+- [JDK 11](https://openjdk.java.net/projects/jdk/11/)
 - [Docker](https://www.docker.com)
 
-*Memory and CPU allocations may need to be increased for successful execution of ccd applications altogether. (On Preferences / Advanced)*
+**Note:** *once docker is installed, increase the memory and CPU allocations (Docker -> Preferences -> Advanced) to the following minimum values for successful execution of ccd applications altogether:*
+
+| Memory   | CPU   |
+| :------: | :---: |
+| 12+ GB   | 6+    |
 
 - [Azure CLI](https://docs.microsoft.com/en-us/cli/azure/install-azure-cli?view=azure-cli-latest) - minimum version 2.0.57
 - [jq Json Processor](https://stedolan.github.io/jq)
+- Mac users, set your default shell to bash `chsh -s /bin/bash`
 
 *The following documentation assumes that the current directory is `ccd-docker`.*
 
 ## Quick start
 
-Checkout `ccd-docker` project:
+1. Checkout `ccd-docker` and `ccd-definition-store-api` projects:
 
 ```bash
 git clone git@github.com:hmcts/ccd-docker.git
 ```
 
-Login to the Azure Container registry:
+2. Login to the Azure Container registry:
 
 ```bash
 ./ccd login
 ```
 Note:
-if you experience any error with the above command, try `az login` first
-
-For [Azure Authentication for pulling latest docker images](#azure-authentication-for-pulling-latest-docker-images)
+if you experience any error with the above command, try `az login` first for [Azure Authentication for pulling latest docker images](#azure-authentication-for-pulling-latest-docker-images)
 
 
 
-Add Postgres V11 DB settings
+3. Add Postgres V11 DB settings - **THIS STEP IS ONLY REQUIRED IF YOU NEED TO MIGRATE TO POSTGRES V11**
 - [Postgres v11 database set-up](/PostgresV11-prerequisites.md)
 
 
 
-Pulling latest Docker images:
+4. Pull latest Docker images:
 
 ```bash
 ./ccd compose pull
 ```
-Running initialisation steps:
+
+
+5. Set up environment: 
 
 Note:
 required only on the first run. Once executed, it doesn't need to be executed again
 
-```bash
-./ccd init
-```
-
-Creating and starting the containers:
+  a. Create docker network
+  ```bash
+  ./ccd init
+  ```
+  
+  b. Export environment variables
+
+  CCD Data Store API and CCD Definition Store API require a set of environment variables which can be set up by executing the following script.
```

Comment:
Can we just say CDM apps (as they include ccd, mca, cpo as well)

Reply:
amended